templates/base/head.tmpl

```diff
@@ -147,7 +147,11 @@
 	<meta property="og:url" content="{{AppUrl}}" />
 	<meta property="og:description" content="{{MetaDescription}}">
 {{end}}
-{{if ne DefaultTheme "gitea"}}
+{{if .IsSigned }}
+	{{ if ne .SignedUser.Theme "gitea" }}
```

Comment:
Could this nested conditional be combined with the one above?

Reply:
It will fail for unauthorizaed users with 
```
template: base/head:150:20: executing "base/head" at <ne .SignedUser.Theme...>: error calling ne: invalid type for comparison
```

Here is what I initially tried:

```html
{{if and .IsSigned (ne .SignedUser.Theme "gitea") }}
		<link rel="stylesheet" href="{{AppSubUrl}}/css/theme-{{.SignedUser.Theme}}.css">
{{else if ne DefaultTheme "gitea"}}
	<link rel="stylesheet" href="{{AppSubUrl}}/css/theme-{{DefaultTheme}}.css">
{{end}}

```

Reply:
Ah yeah, that’s exactly what I was thinking. that’s weird it fails that way,  I would’ve figured that once the first conditional failed it would skip to the next one.